pkg/deploy/controller/deploymentconfig/metrics.go

```diff
@@ -0,0 +1,22 @@
+package deploymentconfig
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// DeploymentConfigControllerSubsystem is how this controller is represented in
+// prometheus metrics.
+const DeploymentConfigControllerSubsystem = "deploymentconfig_controller"
+
+var (
+	deploymentConditionCounter = prometheus.NewGaugeVec(
```

Comment:
@smarterclayton not sure if this should be gauge or counter (i decrease the number when we remove the condition)